# Let's get started with importing

In [2]:
import pandas as pd
import seaborn as sns
import numpy as np
import scipy
from scipy import stats
from statsmodels.stats.multicomp import pairwise_tukeyhsd
from statsmodels.stats.multicomp import MultiComparison

In [3]:
AACdata = pd.read_csv("C:/Users/smkri/Documents/GitHub/DataAfterDarkFinal/Data/aac_intakes_outcomesCLEAN.csv")

In [4]:
AACdata.head()

animal_id_intake animal_type intake_condition      intake_type  \
0          A006100         Dog           Normal            Stray   
1          A006100         Dog           Normal    Public Assist   
2          A006100         Dog           Normal    Public Assist   
3          A047759         Dog           Normal  Owner Surrender   
4          A134067         Dog          Injured    Public Assist   

  sex_upon_intake  intake_year animal_id_outcome outcome_subtype  \
0   Neutered Male         2017           A006100             NaN   
1   Neutered Male         2014           A006100             NaN   
2   Neutered Male         2014           A006100             NaN   
3   Neutered Male         2014           A047759         Partner   
4   Neutered Male         2013           A134067             NaN   

      outcome_type sex_upon_outcome  outcome_year  
0  Return to Owner    Neutered Male          2017  
1  Return to Owner    Neutered Male          2014  
2  Return to Owner    Neutered Male          2014  
3         Transfer    Neutered Male          2014  
4  Return to Owner    Neutered Male          2013

## Followed by some data wrangling

In [16]:
# AACdata = AACdata.drop("outcome_subtype", axis=1)
# AACdata = AACdata.drop("animal_id_outcome", axis=1)

## Now a bit of recoding

In [13]:
def Euth(series):
    if series == "Euthanasia":
        return 0
    if series != "Euthanasia":
        return 1

AACdata["OutcomeR"] = AACdata["outcome_type"].apply(Euth)

In [17]:
AACdata.head()

animal_id_intake animal_type intake_condition      intake_type  \
0          A006100         Dog           Normal            Stray   
1          A006100         Dog           Normal    Public Assist   
2          A006100         Dog           Normal    Public Assist   
3          A047759         Dog           Normal  Owner Surrender   
4          A134067         Dog          Injured    Public Assist   

  sex_upon_intake  intake_year     outcome_type sex_upon_outcome  \
0   Neutered Male         2017  Return to Owner    Neutered Male   
1   Neutered Male         2014  Return to Owner    Neutered Male   
2   Neutered Male         2014  Return to Owner    Neutered Male   
3   Neutered Male         2014         Transfer    Neutered Male   
4   Neutered Male         2013  Return to Owner    Neutered Male   

   outcome_year  OutcomeR  
0          2017         1  
1          2014         1  
2          2014         1  
3          2014         1  
4          2013         1

In [10]:
AACdata.OutcomeR.value_counts()

1    71928
0     2977
Name: OutcomeR, dtype: int64

## --- Out of 74,905 intakes, 2,977 of them were euthanized between 2013-2017.

## Now let's look at those numbers by year

In [16]:
# Separating out the data by year
AAC2013 = AACdata[(AACdata.intake_year == 2013)]
AAC2014 = AACdata[(AACdata.intake_year == 2014)]
AAC2015 = AACdata[(AACdata.intake_year == 2015)]
AAC2016 = AACdata[(AACdata.intake_year == 2016)]
AAC2017 = AACdata[(AACdata.intake_year == 2017)]

In [33]:
# Counting up the values
AAC2013Counts = pd.DataFrame(AAC2013.OutcomeR.value_counts())
AAC2014Counts = pd.DataFrame(AAC2014.OutcomeR.value_counts())
AAC2015Counts = pd.DataFrame(AAC2015.OutcomeR.value_counts())
AAC2016Counts = pd.DataFrame(AAC2016.OutcomeR.value_counts())
AAC2017Counts = pd.DataFrame(AAC2017.OutcomeR.value_counts())

In [57]:
AAC2013Counts.head()

OutcomeR
1      3679
0       261

In [56]:
AAC2017Counts.head()

OutcomeR
1     15965
0       342

In [ ]:
# There is a significant change between intakes for 2013 and 2017, but the Euthanasia rates didn't change as drastically.
# Since the data is older and doesn't line up with our SAC data, I will leave this here.